raiden/api/python.py

```diff
@@ -563,15 +568,37 @@ def get_network_events(self, from_block, to_block):
             to_block=to_block,
         )
 
-    def get_channel_events(self, channel_address, from_block, to_block):
+    def get_channel_events(self, channel_address, from_block, to_block=None):
         if not isaddress(channel_address):
             raise InvalidAddress(
                 'Expected binary address format for channel in get_channel_events'
             )
-        return get_all_netting_channel_events(
+        returned_events = get_all_netting_channel_events(
             self.raiden.chain,
             channel_address,
             events=ALL_EVENTS,
             from_block=from_block,
             to_block=to_block,
         )
+        raiden_events = self.raiden.transaction_log.get_events_in_block_range(
+            from_block=from_block,
+            to_block=to_block
+        )
+        # Here choose which raiden internal events we want to expose to the end user
+        for event in raiden_events:
+            is_user_transfer_event = (
+                isinstance(event[3], EventTransferSentSuccess) or
+                isinstance(event[3], EventTransferSentFailed) or
+                isinstance(event[3], EventTransferReceivedSuccess)
```

Comment:
This:
`isinstance(event[3], (EventTransferSentSuccess, EventTransferSentFailed, EventTransferReceivedSuccess))`
does the same test and it may be a little easier on the eye, just a suggestion.